beetmoverscript/script.py

```diff
@@ -12,16 +12,16 @@
 import aiohttp
```

Comment:
Bleah, apologies for being lazy and not check. I guess it makes sense though "signing-script", "balrog-script", "beetmover-script". I'll add a new issue in balrogscript to correct it and align it consistenly with others, pending the follow-up packaging.